src/policy/packages.h

```diff
@@ -14,6 +15,7 @@
 static constexpr uint32_t MAX_PACKAGE_COUNT{25};
 /** Default maximum total virtual size of transactions in a package in KvB. */
 static constexpr uint32_t MAX_PACKAGE_SIZE{101};
+static_assert(MAX_PACKAGE_SIZE * 4 * 1000 >= MAX_STANDARD_TX_WEIGHT);
```

Comment:
Can you switch the magic constant number of 4 by `WITNESS_SCALE_FACTOR` ?

It won't be required if #22097 gets first, but I don't care about the order, I can take the change there.